README.md

```diff
@@ -199,9 +199,6 @@ export async function getStaticProps() {
   const fetchUrl = `https://graphql.contentful.com/content/v1/spaces/${process.env.NEXT_PUBLIC_CONTENTFUL_SPACE_ID}`;
 
   const fetchOptions = {
-    spaceID: process.env.NEXT_PUBLIC_CONTENTFUL_SPACE_ID,
-    accessToken: process.env.NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN,
-    endpoint: fetchUrl,
     method: "POST",
     headers: {
       Authorization: "Bearer " + process.env.NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN,
```

Comment:
Also wondering if we should change this to a template string since we do that in the other examples.